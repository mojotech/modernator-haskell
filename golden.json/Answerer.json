{
    "seed": -8.556275949528894452e18,
    "samples": [
        {
            "answererId": 4.0,
            "name": "Q+(;³xl\u0001|/I½Ò",
            "sessionId": -28.0
        },
        {
            "answererId": -30.0,
            "name": "\u001fàC\u0003AÛg\u0012×v;l,c\u0016B*þ\u000cX",
            "sessionId": -14.0
        },
        {
            "answererId": -1.0,
            "name": "Ï!W\u001a",
            "sessionId": -15.0
        },
        {
            "answererId": 1.0,
            "name": "ac\u0004\n4¶r)Ü2gJyL|\u0008\u0014ú\u0006\u0007Q\\D|",
            "sessionId": -2.0
        },
        {
            "answererId": -19.0,
            "name": "9d\\$?\u0005>J0ãìI³",
            "sessionId": 15.0
        },
        {
            "answererId": -12.0,
            "name": "\u0002_&tüHzÌ\u00168",
            "sessionId": 19.0
        },
        {
            "answererId": -13.0,
            "name": "pA.ð,\u0014\u001f1\u001a\u000fc",
            "sessionId": -7.0
        },
        {
            "answererId": -26.0,
            "name": "\u0019\u0008C:\u0010\u0006pyT\u0015m¬àZ",
            "sessionId": -25.0
        },
        {
            "answererId": -11.0,
            "name": "\u0015V$Ü{\u0008ö\"k5ç\u001bÓ",
            "sessionId": 24.0
        },
        {
            "answererId": 6.0,
            "name": "\u000eJU1\u0019;åïr\rr+q|",
            "sessionId": -25.0
        },
        {
            "answererId": -20.0,
            "name": "xvi{AÚ7ÅºND\u0007a",
            "sessionId": -11.0
        },
        {
            "answererId": 24.0,
            "name": "\u0015\u0013Y+,x\u0014",
            "sessionId": 11.0
        },
        {
            "answererId": 16.0,
            "name": "\u000eùøQ\u000f\u0007\u0012v\t~V*\"\u0016",
            "sessionId": -2.0
        },
        {
            "answererId": -11.0,
            "name": "#{¬RlzÎI9Kè,\u0019.=Aa<så\u0017&tz?\u0001",
            "sessionId": 19.0
        },
        {
            "answererId": 27.0,
            "name": "Z)¡u\u0019`6ub S¤y3(b\u0019LH/1{0%U\u000em>",
            "sessionId": 15.0
        },
        {
            "answererId": -25.0,
            "name": "WZ\u0014·\u0019(w",
            "sessionId": -23.0
        },
        {
            "answererId": -15.0,
            "name": "\u00056%!\u001dè)\u0006g«:",
            "sessionId": 16.0
        },
        {
            "answererId": 28.0,
            "name": ")VÀugM6·5íw",
            "sessionId": 3.0
        },
        {
            "answererId": -19.0,
            "name": "",
            "sessionId": 11.0
        },
        {
            "answererId": -15.0,
            "name": "N¾8I,¯x«ê\u0014þ\u0004\tÕ\u0001.t\tßÑDK£¨!ylV",
            "sessionId": -21.0
        },
        {
            "answererId": 17.0,
            "name": "g\u001d\u0010\u0003G\u000e5Ij\u000e/\u0015EÂw-TA",
            "sessionId": -14.0
        },
        {
            "answererId": 28.0,
            "name": "6",
            "sessionId": -19.0
        },
        {
            "answererId": 19.0,
            "name": "\u0013\u000f\u0004Üô$8ÈL\u000e4\u0006Í\u0007ÄÛ\u00071yYïô,­Ó\u0013O",
            "sessionId": 5.0
        },
        {
            "answererId": 8.0,
            "name": "T\u000fw\u0000\u001c\u0004I\u0011H+\u0005'\rÎPAw\u0006\u0013=h\u000e",
            "sessionId": -7.0
        },
        {
            "answererId": 29.0,
            "name": "Y¡CmM;\u0019\u00010½*ZrW#2Ð",
            "sessionId": 9.0
        },
        {
            "answererId": 20.0,
            "name": "ò2=B\u0014",
            "sessionId": 2.0
        },
        {
            "answererId": 11.0,
            "name": "\u0016(v²\u0006\u0014J",
            "sessionId": -30.0
        },
        {
            "answererId": -26.0,
            "name": "`~\u0008²Ë¹e",
            "sessionId": -29.0
        },
        {
            "answererId": -11.0,
            "name": "\u0014VÿIh\u001c\u0016\t¨$7ð",
            "sessionId": 12.0
        },
        {
            "answererId": -15.0,
            "name": "z~\u0008fREm\u0015\u0018iðd,XyÃ\u0011_\u0014*",
            "sessionId": 21.0
        },
        {
            "answererId": -12.0,
            "name": "*©õzc\u001e}»\u0019\u000cÞë<",
            "sessionId": 8.0
        },
        {
            "answererId": -8.0,
            "name": "&Ö-ZÃa*S%RQ¹{ª\u0013k1k>",
            "sessionId": -20.0
        },
        {
            "answererId": -19.0,
            "name": "¾8ò\u001e+¶@Û\u0006'rÕ\u0001\u001a£@",
            "sessionId": 30.0
        },
        {
            "answererId": -20.0,
            "name": "êÍ÷+\u000b\u001fZ|E\"\u0018\u0015û® LZ=É\u0011r|",
            "sessionId": 17.0
        },
        {
            "answererId": -21.0,
            "name": "N-t¹D)\u0010Ï²\u0019 ¥Î\u0019z¬L7\rU",
            "sessionId": -12.0
        },
        {
            "answererId": 9.0,
            "name": "J\u000e*½~Û-*",
            "sessionId": 19.0
        },
        {
            "answererId": 3.0,
            "name": "\u000fc«ÅÍLâ|Ä",
            "sessionId": -5.0
        },
        {
            "answererId": 1.0,
            "name": "&\u001b\u001dOáümãT:\\,²x\u0006\\lRP^#7Wn",
            "sessionId": 7.0
        },
        {
            "answererId": 19.0,
            "name": ",w?÷:³ØFu{E¨,^\u0006TP¦+.Ù",
            "sessionId": -11.0
        },
        {
            "answererId": 11.0,
            "name": "ÅK*dë<",
            "sessionId": 20.0
        },
        {
            "answererId": 23.0,
            "name": "\u001aënu< M\u000b[\rs¢¼|b>\teðïCnsÇà\u001eZ\u001a\u001e",
            "sessionId": 25.0
        },
        {
            "answererId": -4.0,
            "name": "vÄTqrÝØ/Dm¾K[f\rJ3$c",
            "sessionId": -13.0
        },
        {
            "answererId": 9.0,
            "name": "ah5\u0004)1I\u001c<\u0002",
            "sessionId": -21.0
        },
        {
            "answererId": 17.0,
            "name": "G!®J",
            "sessionId": 17.0
        },
        {
            "answererId": 25.0,
            "name": "\u000efZ>u\u0015\u0002\u0005ýÉ\u0008\\dÃ\\\u000bGÂ\u0000\u0004M6\u0008UEZ",
            "sessionId": 2.0
        },
        {
            "answererId": -3.0,
            "name": "#Go%aÜ\u001cf\u001dù¨iì@±/ÕÊx|\u000cW6\u000f",
            "sessionId": 15.0
        },
        {
            "answererId": 2.0,
            "name": "¬:L&7XE3%áAX:¸,éN$:0{îg",
            "sessionId": 23.0
        },
        {
            "answererId": -25.0,
            "name": " S\u000e\u0008\u001dé¨geRG\u0016\u0006ï5\u0005µ\u001c\u0016J",
            "sessionId": 17.0
        },
        {
            "answererId": -25.0,
            "name": "cäý,p¥VL²D÷oFbÚ¢Ô\u0013¼\u0013Ut\u000cV",
            "sessionId": 7.0
        },
        {
            "answererId": 2.0,
            "name": "mkz_ø/Ý[EC!\u001e@\u0013¦×",
            "sessionId": 11.0
        },
        {
            "answererId": -3.0,
            "name": "}%$7Êë\u001b;R`",
            "sessionId": -11.0
        },
        {
            "answererId": -3.0,
            "name": "'J!¶ÿÖxg¹¾]ntKCcZïÉ%Oc",
            "sessionId": -1.0
        },
        {
            "answererId": 10.0,
            "name": "]/!bç\u0002QE8I/AFGî¿!\u001e\n\u0013l",
            "sessionId": 30.0
        },
        {
            "answererId": -2.0,
            "name": "V\u0000½9B\u0010w<ôC\"3krPg\u0006",
            "sessionId": 28.0
        },
        {
            "answererId": -26.0,
            "name": "\u0006l5#] N}g\u0003",
            "sessionId": 29.0
        },
        {
            "answererId": -3.0,
            "name": "Ø²\u00005Fzâ]\u0016Kñ\u0008Y",
            "sessionId": 20.0
        },
        {
            "answererId": -27.0,
            "name": "t!cçi¬1\u0010\u0004(M0\u0019²=\u0002\u0000h$",
            "sessionId": 11.0
        },
        {
            "answererId": -9.0,
            "name": "\u0003úB\u0011\u001c\u0003<RO^\u001b",
            "sessionId": 7.0
        },
        {
            "answererId": 14.0,
            "name": "ÓL\u0005\u000f\u000e\u0006",
            "sessionId": -1.0
        },
        {
            "answererId": -11.0,
            "name": "\u0010£\u001c+)p/\u001db/^ßXrOXRO\u0011D\\\u001b\"~'Âr",
            "sessionId": 5.0
        },
        {
            "answererId": -9.0,
            "name": "r\u001d\u0013S\u001eTÕY°R'FñJ¿³UÈ",
            "sessionId": -23.0
        },
        {
            "answererId": -13.0,
            "name": "8TBc9\n6ªLÝC\u000el\r-\u000c\u0000",
            "sessionId": 17.0
        },
        {
            "answererId": 8.0,
            "name": "Yqú",
            "sessionId": 14.0
        },
        {
            "answererId": 21.0,
            "name": "e#*\u0004z\u000feÏWS^\u0002",
            "sessionId": 10.0
        },
        {
            "answererId": -2.0,
            "name": "ÌY(N\\ÛÐuow@",
            "sessionId": 28.0
        },
        {
            "answererId": -21.0,
            "name": "ktF_2ÅË%a\u001a& ÓÓAÌ",
            "sessionId": 10.0
        },
        {
            "answererId": 6.0,
            "name": "m\u001eª[qËL\u0016¥9ï)~mD'2\u0017\u0018Ã¨FÒØ:\u000b]s",
            "sessionId": 22.0
        },
        {
            "answererId": -16.0,
            "name": "\u0012O=ZtIô%|\u001c\u001fä",
            "sessionId": 23.0
        },
        {
            "answererId": -8.0,
            "name": "u, \u0008låa\"gA|[Öq9)\u0019=A54~\u0011fe¦",
            "sessionId": -8.0
        },
        {
            "answererId": 14.0,
            "name": "ZÝ}\u0015]]\u0007\u000f,4´60",
            "sessionId": -3.0
        },
        {
            "answererId": 19.0,
            "name": "`Xû¸\u000f\u000b,ºÿ\u001dÔN,\u0017,",
            "sessionId": 4.0
        },
        {
            "answererId": 19.0,
            "name": "_\u0005kô\u0015%KSçæ\u00154aE.\"5{l\u0008\"\u001b~\u0007\u000f",
            "sessionId": 6.0
        },
        {
            "answererId": -22.0,
            "name": "R4bBv/\u0012j`\n\u001d]ùhþÄñÅ7r¦û,",
            "sessionId": 15.0
        },
        {
            "answererId": -14.0,
            "name": "×ìN\u001d6=\u0001×8'8\u001a\u001f\u0018\u0018\raØ\u001c",
            "sessionId": -1.0
        },
        {
            "answererId": 16.0,
            "name": "1",
            "sessionId": 13.0
        },
        {
            "answererId": -26.0,
            "name": "2OYZ;]xv(1,-!;",
            "sessionId": -15.0
        },
        {
            "answererId": -30.0,
            "name": "«s.ªw¨.\u0015",
            "sessionId": -6.0
        },
        {
            "answererId": -2.0,
            "name": "<0O\u0003Lu\u0010%&t\u0015o]\u0016jo",
            "sessionId": 12.0
        },
        {
            "answererId": -14.0,
            "name": "Rp.b7\u0003Ä\u001d:E4n:\u0017n",
            "sessionId": 0.0
        },
        {
            "answererId": -20.0,
            "name": "r\u001aa@\u0018\u001c»\u001b^ \u0012g\u001eM\r\u001a«n`Y3*é\"=\u0010",
            "sessionId": -27.0
        },
        {
            "answererId": -17.0,
            "name": "Ý\u0019ÂMÃN\u001c\u0011\u0002",
            "sessionId": -17.0
        },
        {
            "answererId": -14.0,
            "name": "<t\u0000Wn\u0012\u000e&°d\u001b ùl¥ç\u000eÃ6G\u0008\n\u0002[xLV",
            "sessionId": 8.0
        },
        {
            "answererId": 22.0,
            "name": "I",
            "sessionId": 11.0
        },
        {
            "answererId": -25.0,
            "name": "1Zò0\u0016û¯",
            "sessionId": 16.0
        },
        {
            "answererId": 9.0,
            "name": "e\r\u0016F\\hd!]E ]\u0015í\u0011eÆLð=cz0AB",
            "sessionId": -10.0
        },
        {
            "answererId": -3.0,
            "name": "\u001c<^3\u001f\u001eS_\u0001D\u0019kR\u000cel3WÇgd:\u0008\u0005c\t!8\u0004Á",
            "sessionId": 4.0
        },
        {
            "answererId": 18.0,
            "name": "\u0006\r 9\r\u0016)6<S\u0007kZ\"ÝH\u0003\u0013&\u0017\u001a#\u0018§m¶",
            "sessionId": -25.0
        },
        {
            "answererId": 12.0,
            "name": "cQÍ¨|1fp»¿\u0010^3:ó\u001bNcú\u001c¯c\u0008÷",
            "sessionId": 25.0
        },
        {
            "answererId": -11.0,
            "name": "\u0016\u00145\u000eC_+jpÁ\u0003",
            "sessionId": -5.0
        },
        {
            "answererId": 3.0,
            "name": "Ò\u0019EK\u001e'",
            "sessionId": -11.0
        },
        {
            "answererId": 27.0,
            "name": "H",
            "sessionId": -17.0
        },
        {
            "answererId": -7.0,
            "name": " Z~,rOT\u000b\u0019\n}e3©f\u001fÉ`:\u0017ýKç",
            "sessionId": 24.0
        },
        {
            "answererId": -7.0,
            "name": "-\u001e\u001dg=Ý",
            "sessionId": 5.0
        },
        {
            "answererId": 14.0,
            "name": "\u0014k_}\u0001]O\u0013\u000e",
            "sessionId": 20.0
        },
        {
            "answererId": 0.0,
            "name": "(øf",
            "sessionId": -1.0
        },
        {
            "answererId": 12.0,
            "name": "\"ù\u000e58\r\"1\"\u000c·w\u0002",
            "sessionId": -13.0
        },
        {
            "answererId": -18.0,
            "name": " t>XcØrÑ7^n\\78mÏ'[Ù÷\"gn=J\u0000o\u0003",
            "sessionId": 7.0
        },
        {
            "answererId": -1.0,
            "name": "­ÃNt°®#.8B¼j^\u0003~t8©T\n\u0013\u0003<B\u0012\u0018",
            "sessionId": 29.0
        },
        {
            "answererId": 26.0,
            "name": "O\rùkòD^¨iP©1}\"YãÁ%\u00026",
            "sessionId": -12.0
        },
        {
            "answererId": 14.0,
            "name": "AEHãú¾ù\u0012iÅ\u0013â8\u0012\u000b",
            "sessionId": 4.0
        },
        {
            "answererId": 9.0,
            "name": "\u0012eÎ\u0012e",
            "sessionId": -12.0
        },
        {
            "answererId": 26.0,
            "name": "\u0018\u000b",
            "sessionId": -25.0
        },
        {
            "answererId": 17.0,
            "name": "'v>]HÔQf\u0005D\u0011CF\u0019s°$k'Êi\u0012#",
            "sessionId": -7.0
        },
        {
            "answererId": -9.0,
            "name": "ey\n&",
            "sessionId": -14.0
        },
        {
            "answererId": 27.0,
            "name": ">i,l\u000cp°jvu",
            "sessionId": -11.0
        },
        {
            "answererId": -5.0,
            "name": "^x¸4vI\u0012+\u001dK\u0013Í¦t^ÅOX¦\u0012D³,JkÄ",
            "sessionId": -16.0
        },
        {
            "answererId": 10.0,
            "name": "",
            "sessionId": 17.0
        },
        {
            "answererId": 12.0,
            "name": "?H4Pg5'O\u0001,9ÊQF0",
            "sessionId": -17.0
        },
        {
            "answererId": 5.0,
            "name": "\u0001nY7Ó¥\\/\u0002\u0018hTn\u0015c\n",
            "sessionId": 26.0
        },
        {
            "answererId": 4.0,
            "name": "m\u001dE\u0004ü¹DF\u0005>CÇV*Ö¸YGKk\u0012bª",
            "sessionId": -11.0
        },
        {
            "answererId": 3.0,
            "name": "(;_,yAû<*;GzÙ",
            "sessionId": -28.0
        },
        {
            "answererId": -25.0,
            "name": "®R!÷'GHs2",
            "sessionId": -28.0
        },
        {
            "answererId": 24.0,
            "name": "~\u000c\u001ah\u001a#",
            "sessionId": -1.0
        },
        {
            "answererId": -21.0,
            "name": "'\u0006º!X\u0002í.BFûO97\u0013Õ\u0019oL\u001c",
            "sessionId": -10.0
        },
        {
            "answererId": 0.0,
            "name": "¨R\u0005*,2~hÔ]2\u0016sÛ\u0018}Sb\u001d#=\u000b\u001ad\u0001=",
            "sessionId": -6.0
        },
        {
            "answererId": -16.0,
            "name": "KÝb?\u0001 ×úê7}WÓCy»|Ònf<",
            "sessionId": -22.0
        },
        {
            "answererId": 22.0,
            "name": "\"Ôw#<Ë_#\u0015²TÙ",
            "sessionId": 29.0
        },
        {
            "answererId": 20.0,
            "name": "UÇ\u001b\u0007{h7jd\u0004Hc¸:]Mu\u0003³4\u0018G",
            "sessionId": 23.0
        },
        {
            "answererId": -17.0,
            "name": "\u001c\u0005*!Ib4\u0007ÊM\u0006ZJb¾é\u001dÛø­i\u001e¢",
            "sessionId": -21.0
        },
        {
            "answererId": 11.0,
            "name": "\"u÷+hç¯",
            "sessionId": -19.0
        },
        {
            "answererId": 4.0,
            "name": "±ÀQ\u0008Ì\u001bEea@J5\u000c2pF\u001eé",
            "sessionId": 26.0
        },
        {
            "answererId": -9.0,
            "name": "y#x\u0017Ï\u0004IX\u000c6",
            "sessionId": 24.0
        },
        {
            "answererId": 16.0,
            "name": "E\u0008ÿ\u0018",
            "sessionId": 10.0
        },
        {
            "answererId": -1.0,
            "name": "\u0011",
            "sessionId": -11.0
        },
        {
            "answererId": -29.0,
            "name": "\u0002~»R\u0000¾\u00118=ã\u0002\u0017½nRbh\u001beîfL-",
            "sessionId": -22.0
        },
        {
            "answererId": -9.0,
            "name": "\u001d\u000b0BYnú\u0007¸x\u0013E9~¡FòYxAÀj",
            "sessionId": 1.0
        },
        {
            "answererId": 5.0,
            "name": "`\u0019FqB¾H.9P",
            "sessionId": -20.0
        },
        {
            "answererId": 23.0,
            "name": ";lmÈÎM\u0019ã\u001a_\u0011Ç",
            "sessionId": 15.0
        },
        {
            "answererId": 12.0,
            "name": "6\u0001É\n\u000cl",
            "sessionId": -19.0
        },
        {
            "answererId": -29.0,
            "name": "\u000bà²",
            "sessionId": -16.0
        },
        {
            "answererId": -6.0,
            "name": "<#1aò1I",
            "sessionId": 4.0
        },
        {
            "answererId": -4.0,
            "name": "K\u0015?nc\u00125È",
            "sessionId": 28.0
        },
        {
            "answererId": 5.0,
            "name": "3a ]Á\u001eÝ\u0006",
            "sessionId": -8.0
        },
        {
            "answererId": -11.0,
            "name": "½%@¢!ÍX=\\í+Øl\u001d½6\u0006S\u000bëK\t1x",
            "sessionId": 21.0
        },
        {
            "answererId": 25.0,
            "name": "Rá\u0010\u000b¯oÇWÛú1~sï9\t·7¿ù",
            "sessionId": 18.0
        },
        {
            "answererId": -10.0,
            "name": "C\u001c\u001eÎ±Óeê\u0019\u0017t:6@©(",
            "sessionId": 30.0
        },
        {
            "answererId": -12.0,
            "name": "ø_\u0019Ä\n%z¶¹",
            "sessionId": 23.0
        },
        {
            "answererId": 29.0,
            "name": "^àZþXXúMjwî",
            "sessionId": -14.0
        },
        {
            "answererId": -11.0,
            "name": "1J\u0002_\r[\u000c\u001dOK_5\u000eå4\\?IL(*zÁ",
            "sessionId": 13.0
        },
        {
            "answererId": 24.0,
            "name": "Ñ&<¯÷\u0001´qó1{Lè-ãqÇXp>?Æ\u0001-FI]",
            "sessionId": -24.0
        },
        {
            "answererId": 21.0,
            "name": "d°>oZ$|\u0006\u0012\u0001H/©7²aY;\u001bM(\u0006%\u0014\tD",
            "sessionId": 0.0
        },
        {
            "answererId": 12.0,
            "name": "¸L\"",
            "sessionId": 13.0
        },
        {
            "answererId": -4.0,
            "name": "<\u0016\u0004k5!Ù\u0015î\u0013Ö3\u001cSá\u0018\u0015",
            "sessionId": -24.0
        },
        {
            "answererId": 6.0,
            "name": "/\r\"*D\u000c.OWi\u0010n\u001a%`6",
            "sessionId": -21.0
        },
        {
            "answererId": 0.0,
            "name": "\u0013\u0003TÒ\u0002\u0013\u0016SO<ËXQ\u000c¯Â4u1",
            "sessionId": 27.0
        },
        {
            "answererId": 27.0,
            "name": "Î¡_",
            "sessionId": 26.0
        },
        {
            "answererId": -21.0,
            "name": "5\u0013\u001d'ÕØÐZ\u0015q·zf¡}Ë7Z\u0003RG\u0016\u001d\u0008\u0014\u0002",
            "sessionId": 16.0
        },
        {
            "answererId": 26.0,
            "name": "*\u0018a(a7\u000eY",
            "sessionId": -20.0
        },
        {
            "answererId": -16.0,
            "name": "~~`L\u000f¥wú$.\\=o5d(Ân®D9#1",
            "sessionId": 20.0
        },
        {
            "answererId": 5.0,
            "name": "L÷LR",
            "sessionId": 4.0
        },
        {
            "answererId": -3.0,
            "name": " Rã",
            "sessionId": -22.0
        },
        {
            "answererId": 24.0,
            "name": "o4BIüàN\u0002c\u0019V",
            "sessionId": 27.0
        },
        {
            "answererId": 11.0,
            "name": "OèTjp\\n\u0016F.\u0008@­\u000e\u0019\u0012ZV`V\u0012-Ø½z+\u0005z",
            "sessionId": -20.0
        },
        {
            "answererId": 26.0,
            "name": "%:IU\u0008a>Ç;m\u001fF¹>d4=(",
            "sessionId": 27.0
        },
        {
            "answererId": -1.0,
            "name": "\u000e\\«ì-r\u0008\u0002\u000c8\\90.â¯\u00151",
            "sessionId": 2.0
        },
        {
            "answererId": 9.0,
            "name": "iM]}4u*R\u0008K9e7\u001aWN",
            "sessionId": 15.0
        },
        {
            "answererId": -16.0,
            "name": "\u0000H&L9õr.\u0000ßO\u0000¬\u0008Å\u000b¶\u0005r",
            "sessionId": -11.0
        },
        {
            "answererId": 4.0,
            "name": "Âg7¢\\",
            "sessionId": -24.0
        },
        {
            "answererId": -1.0,
            "name": "\u0018VAð6!",
            "sessionId": -4.0
        },
        {
            "answererId": 23.0,
            "name": "R9èQl¬\u00049",
            "sessionId": -20.0
        },
        {
            "answererId": 0.0,
            "name": "oü\u00061-n\u0011\u000f\u0004#Ñ\u0004A}\r*2ºs¦\u0016j\u0000n",
            "sessionId": 24.0
        },
        {
            "answererId": 10.0,
            "name": "£c",
            "sessionId": 12.0
        },
        {
            "answererId": 25.0,
            "name": "\u0010å:X\\â·j\u0006¦tX(X+m)'f",
            "sessionId": -21.0
        },
        {
            "answererId": -19.0,
            "name": "\u0003\u001f\u000b}RvÅS],&s,='z",
            "sessionId": -18.0
        },
        {
            "answererId": -7.0,
            "name": "di/EFûÞ7_C2)[\u000c",
            "sessionId": -30.0
        },
        {
            "answererId": -24.0,
            "name": "\u0003Kc\r\u0019\u001bK³M\u001fEYiÝ£\t|U4\u0013-\n",
            "sessionId": -4.0
        },
        {
            "answererId": 15.0,
            "name": "Í,#",
            "sessionId": 0.0
        },
        {
            "answererId": -2.0,
            "name": "m\u0017¬\u0016X\u0016æè9-²f¾\u0007SÅJÆÁBB\u000eh%õ",
            "sessionId": -5.0
        },
        {
            "answererId": -26.0,
            "name": "\u001en°W\u001e£jr\u000bS\u0018Õ×\u0019\u001bt\u000btL(5Û",
            "sessionId": -4.0
        },
        {
            "answererId": -29.0,
            "name": "=ëñI%7UïZ~F·;\u0007Xò",
            "sessionId": -25.0
        },
        {
            "answererId": -7.0,
            "name": "¨\u000f\u0011>$®4ì\u000f\u0003",
            "sessionId": 6.0
        },
        {
            "answererId": 19.0,
            "name": "\u0006FOL¦ß\u00121]lg\rhT!E",
            "sessionId": -8.0
        },
        {
            "answererId": 22.0,
            "name": ":\u001e)k\u001aÈ",
            "sessionId": 30.0
        },
        {
            "answererId": 9.0,
            "name": "\u0003v´\u000eEÖ¡¯ù¶°ä",
            "sessionId": -29.0
        },
        {
            "answererId": -16.0,
            "name": ".\u001a;a\u0007Ý\u001cvCâSC°\u000e\u0007`\u000e8f8!â\u0003N3÷,",
            "sessionId": -13.0
        },
        {
            "answererId": -6.0,
            "name": "IÎK\u000erC\u0003ûm\u0017C|ì",
            "sessionId": 26.0
        },
        {
            "answererId": 21.0,
            "name": "/Ò?`\u0005PYvQ¶¬WO\u0003\u0012t\u0015»p\u000b9T%",
            "sessionId": -26.0
        },
        {
            "answererId": -3.0,
            "name": "OKRb",
            "sessionId": 25.0
        },
        {
            "answererId": 25.0,
            "name": "ª\u0005l4\u0007\u0019kZÀ\u001e\u000e\u000e>9\u0014}|",
            "sessionId": 7.0
        },
        {
            "answererId": -15.0,
            "name": "\n\u00062\u0006\u0001Nt\u0011MVi\u00109r=",
            "sessionId": 30.0
        },
        {
            "answererId": -14.0,
            "name": "QËBUl|1\u00057\u0004\u0002_\u0012-j\u0013,¨+\u0013ásu>",
            "sessionId": -15.0
        },
        {
            "answererId": -27.0,
            "name": "â\u0010 wH´ñê©NaW\"j\u001b6>Ò\u000bæITY",
            "sessionId": -28.0
        },
        {
            "answererId": -30.0,
            "name": "ÄÚl*k\u00142,9\u001d\n",
            "sessionId": 21.0
        },
        {
            "answererId": 3.0,
            "name": "¨RTSÜ}\u001a&\u0007 AEË",
            "sessionId": 8.0
        },
        {
            "answererId": -29.0,
            "name": "IZ\":<\u0003È½¾1\u0018v,E{",
            "sessionId": 21.0
        },
        {
            "answererId": 7.0,
            "name": "P9\u0013%",
            "sessionId": -23.0
        },
        {
            "answererId": 7.0,
            "name": "t+\u0008áyF:O\u0002BC(-³",
            "sessionId": -5.0
        },
        {
            "answererId": -1.0,
            "name": "\u0016c=.l\u0004ô\u0003f-ÍhQ",
            "sessionId": 15.0
        },
        {
            "answererId": 23.0,
            "name": "á\u000e(\u000c?!#~ÓbäÑ\u0004SY\u0014(ý\u001a\u0018 Ø%pã\u001f",
            "sessionId": -3.0
        },
        {
            "answererId": -22.0,
            "name": "\u0019\u001dv\u0016G9*àN%E1os7",
            "sessionId": 1.0
        },
        {
            "answererId": 21.0,
            "name": "W6",
            "sessionId": 13.0
        },
        {
            "answererId": -26.0,
            "name": "½B{Ó\u0003%9JIoy}<\u0001PHê\u000e'.ài\u0000\u0006",
            "sessionId": -12.0
        },
        {
            "answererId": -14.0,
            "name": "3V\u00081ÿ+\u000fkñ\u001a¡0%\\I\u000bs]Mù(yÜxTØ\u0016",
            "sessionId": 0.0
        },
        {
            "answererId": 26.0,
            "name": "E)4",
            "sessionId": -27.0
        },
        {
            "answererId": -16.0,
            "name": "L\u000eÒÇI\u0018Mbq1=\u0011\u0003$\u000b;ö{\\\u0007:Þ!g",
            "sessionId": 18.0
        },
        {
            "answererId": -30.0,
            "name": "çvÊT>\"g\u0001\u0004^\u0006´R\u001a¾äÿ\u0010ë\u0007\u0013\u0003=\u0003Q",
            "sessionId": 0.0
        },
        {
            "answererId": 22.0,
            "name": "j³@",
            "sessionId": -2.0
        },
        {
            "answererId": -2.0,
            "name": "£j_¤8E\u001a\u0004e?]/\u001d",
            "sessionId": 7.0
        },
        {
            "answererId": 14.0,
            "name": "&\u001d?\u0018þ]-]æX\u0004!S<s:{{Ö",
            "sessionId": -3.0
        },
        {
            "answererId": -22.0,
            "name": "\u0016j",
            "sessionId": 3.0
        }
    ]
}