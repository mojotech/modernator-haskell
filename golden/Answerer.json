{
    "seed": 7998112267436432188,
    "samples": [
        {
            "answererId": 15,
            "name": "E<Ö3\t\u0001_\u0012ý%?A&Ã\u000evCCätj\nH\u0016-\u0011ÔO",
            "sessionId": -4
        },
        {
            "answererId": -1,
            "name": "â_",
            "sessionId": 6
        },
        {
            "answererId": -17,
            "name": "",
            "sessionId": -10
        },
        {
            "answererId": -17,
            "name": "°\u0018\u000b7Vf$",
            "sessionId": 22
        },
        {
            "answererId": 16,
            "name": "\\\u0005¿a\u0001\u0003M\u0019\u0016`Q",
            "sessionId": -10
        }
    ]
}